crates/bevy_light/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "bevy_light"
+version = "0.17.0-dev"
+edition = "2024"
+description = "Keeps the lights on at Bevy Engine"
```

Comment:
Playful but I think a more share description could be made. :)

Reply:
open to suggestions :)